### PR TITLE
Register seldom.is-a.dev

### PR DESCRIPTION
**Pull request by seldomgreeen**
Register seldom.is-a.dev with CNAME record pointing to seldomgreen.me.

### CODE REVIEW

File: domains/seldomgreen.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "seldomgreeen",
+           "email": "72107155+seldomgreeen@users.noreply.github.com",
+           "discord": "761646485334523954"
+        },
+    
+        "record": {
+            "CNAME": "is-a.dev"
```
Comment (DEV-DIBSTER) [inline domains/seldomgreen.json:9]: Could you provide the reason as to why you want this domain to point to our homepage?

File: domains/seldomgreen.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "seldomgreeen",
+           "email": "72107155+seldomgreeen@users.noreply.github.com",
```
Comment (DEV-DIBSTER) [inline domains/seldomgreen.json:4]: Invalid email, we need a email that we are able to contact you, not a no reply email.

File: domains/seldom.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "seldomgreeen",
+           "email": "72107155+seldomgreeen@users.noreply.github.com",
```
Comment (DEV-DIBSTER) [inline domains/seldom.json:4]: Invalid email, we need a email that we are able to contact you, not a no reply email.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Review by seldomgreeen — COMMENTED** [review]
I want the email to be seldom@seldomgreen.me

**Pull request closed without merging**